platform-tools-sdk/cargo-test-sbf/src/main.rs

```diff
@@ -171,7 +171,9 @@ fn test_solana_package(
     );
 
     // Pass --sbf-out-dir along to the solana-program-test crate
-    env::set_var("SBF_OUT_DIR", sbf_out_dir);
+    unsafe {
+        env::set_var("SBF_OUT_DIR", sbf_out_dir);
+    }
```

Comment:
This one is necessary to execute Solana tests. Many functions throughout the monorepo look for this env var to obtain where the binaries are.

`cargo-test-sbf` is supposed to be a wrapper to invoke those testing functions.